src/UiPath.Workflow/Activities/ScriptingJitCompiler.cs

```diff
@@ -73,7 +74,12 @@ public override LambdaExpression CompileExpression(ExpressionToCompile expressio
                 .Select(GetTypeName));
         var finalCompilation = compilation.ReplaceSyntaxTree(syntaxTree, syntaxTree.WithChangedText(SourceText.From(
             CreateExpressionCode(types, names, expressionToCompile.Code))));
-        var results = ScriptingAotCompiler.BuildAssembly(finalCompilation);
+        
+        var collectibleAlc = new AssemblyLoadContext("ScriptingJit" + Guid.NewGuid(), true);
```

Comment:
Do you need to explicitly unload the AssemblyLoadContext?

Reply:
We don't need to, as it should get garbage collected away (along with its assemblies) once there are no strong references to any assembly loaded in that context.

However, there is a caveat here - some WPF services (see `SafeSecurityHelper.cs` in multiple WPF assemblies) hold on to these collectible assemblies (the logic is there for them to not hold on to dynamic assemblies, but not for collectible ones), but in Studio we do some reflection magic to clear collectible assemblies from these Safe Security Helpers once in a while, so in practice this should be solved.